services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -1148,8 +1148,16 @@ func (s *State) DeleteQueuedVersionIfExists(environment string, application stri
 	return s.DeleteQueuedVersion(environment, application)
 }
 
-func (s *State) GetEnvironmentApplicationVersion(environment, application string) (*uint64, error) {
-	return s.readSymlink(environment, application, "version")
+func (s *State) GetEnvironmentApplicationVersion(ctx context.Context, transaction *sql.Tx, environment, application string) (*uint64, error) {
```

Comment:
We should keep this functionality for reading from the manifest for kuberpult deployments that don't use the database

Reply:
Why would the manifest-service need to read from the manifest-repo? What is the use-case?

Reply:
Ah, wrong service, my bad